Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/groovy
+
+// Copyright (C) 2019 Luxoft Sweden AB
+
+def image
+
+node("Component") {
+    stage("Clean up the environment") {
+        sh '''
+        rm -rf *
+        '''
+    }
+
+    stage("Checkout") {
+        checkout scm
+    }
+
+    stage("Build Docker container") {
+        image = docker.build("template-dbus-service", "-f Dockerfile .")
+    }
+
+    image.inside {
+        stage("Download Raspberry Pi SDK") {
+            sh '''
+            wget https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY_rpi-qtauto/lastSuccessfulBuild/artifact/artifacts_rpi-qtauto/pelux-glibc-x86_64-core-image-pelux-qtauto-neptune-dev-cortexa7t2hf-neon-vfpv4-toolchain-*.sh/*zip*/artifacts_rpi-qtauto.zip
+            unzip artifacts_rpi-qtauto.zip
+            '''
+        }
+
+        stage("Install Raspberry Pi SDK") {
+            sh '''
+            chmod +x pelux-glibc-x86_64-*.sh
+            sudo ./pelux-glibc-x86_64-*.sh -y -d /opt/pelux-sdk-x86_64/master
```

Comment:
Perhaps we should not install the SDK in /opt? I assume that is an overlay2 fs and I assume we could potentially have the same problems as https://github.com/Pelagicore/pelux-manifests/pull/234 fixed in pelux-manifests?

How does image.inside() work? I assume the workspace directory is bind mounted into the container since it is available when why build below. Can the SDK be installed there? Perhaps we should checkout the source in a subdir in the workspace dir and install the SDK in another subdir?